xknx/core/state_updater.py

```diff
@@ -209,9 +209,8 @@ async def connection_state_change_callback(
         if state == XknxConnectionState.CONNECTED:
             if not self.started:
                 self._start()
-        else:
-            if self.started:
-                self._stop()
```

Comment:
🧑‍💻

Reply:
I don't entirely agree here, but if you want I can change it back to the original.

Reply:
Nah, that's ok. I wasn't sure about that one either.